config/database.yml

```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: db
+  username: postgres
+  password: password
```

Comment:
these could probably be set with the DATABASE_URL env var in the compose file instead of hard coded this will make it easier to customise in other environments and means this file wouldn't need to be changed at all as part of this pr

Reply:
This has been done!